app/controllers/settings/deletes_controller.rb

```diff
@@ -42,7 +42,7 @@ def challenge_passed?
   end
 
   def destroy_account!
-    current_account.suspend!
+    current_account.suspend!(origin: :local)
```

Comment:
Hmm thinking about that, a self-deletion being displayed as a suspension might be misleading and problematic. Though that's a permanent suspension so I guess it should be fine, just returning 410 with a generic “Gone” page.